_pages/opportunity-grants.md

```diff
@@ -58,7 +58,7 @@ You can apply for 3 types of grant:
 - Visa application fee: reimbursement for the cost of a UK Standard Visitor visa (up to £100).
 
 {% comment %}
-To apply, please fill out [this form](https://2023.djangocon.eu/opportunity-grants/).
+To apply, please fill out [this form]({{ site.opportunity_grant_application }}).
```

Comment:
I’d prefer we use link text descriptive of the target page so screen reader users can navigate link-by-link, doesn’t have to be this exact text if you’d rather use something else:

```suggestion
To apply, please fill out our [Opportunity Grant Application]({{ site.opportunity_grant_application }}) form.
```